src/Components/Form/FormFields/FormField.tsx

```diff
@@ -71,7 +71,9 @@ const FormField = ({
         )}
       </div>
       <div className={field?.className}>{props.children}</div>
-      <FieldErrorText error={field?.error} className={field?.errorClassName} />
+      {field?.error && (
+        <FieldErrorText error={field.error} className={field?.errorClassName} />
+      )}
```

Comment:
use `errorClassName="hidden"` instead. This change will now cause layout shift everytime an error appears or disappears which is not ideal.

Reply:
I was just irritated with how it was talking up space under the input, hidden works;

Reply:
then let's switch to `hidden`. looks like this currently

<img width="1087" alt="image" src="https://github.com/user-attachments/assets/21d89323-de32-4a80-b825-cd9be0cd24ae">

Reply:
> use `errorClassName="hidden"` instead. This change will now cause layout shift everytime an error appears or disappears which is not ideal.

added

Reply:
> then let's switch to `hidden`. looks like this currently
> 
> <img alt="image" width="1087" src="https://private-user-images.githubusercontent.com/25143503/378273374-21d89323-de32-4a80-b825-cd9be0cd24ae.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mjk3NTMyOTQsIm5iZiI6MTcyOTc1Mjk5NCwicGF0aCI6Ii8yNTE0MzUwMy8zNzgyNzMzNzQtMjFkODkzMjMtZGUzMi00YTgwLWI4MjUtY2Q5YmUwY2QyNGFlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDI0VDA2NTYzNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE4YzA4MmUxOTlkNWNhMzMxZTk4ZThmY2JlMmJiZDUzNGZjOGJlYzczMmQzYjhiZTJhOWZlZjAxMzYzNzY1ODUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.0ir7QczfUlz6loxBH0Ti_OzRz76dyaWhz8yW1BC_I-Y">

@rithviknishad   from which page you have taken this screenshot ,actually in my system  flag emoji is not coming it is showing 'IN' in the ph number input field

Reply:
Assets > Select an asset > Update

Reply:
This issue is not resolved

Reply:
> This issue is not resolved

I'm sorry, I added errorClassName="hidden" but didn't check whether it works or not. I will fix that